browser/components/shared/constants/align.js

```diff
@@ -0,0 +1,7 @@
+const Align = {
+    LEFT: 'left',
```

Comment:
I know the code I sent you was in "all-caps".
The other component enums are in Titlecase, which I like better. At least they should all follow the same convention.
```
{
    Left: 'left',
    Center: 'center',
    Right: 'right'
}
```